src/Core/Addon/Module/ModuleManager.php

```diff
@@ -294,7 +294,9 @@ public function install($source)
         if (!empty($source)) {
             $this->moduleZipManager->storeInModulesFolder($source);
         } elseif (!$this->moduleProvider->isOnDisk($name)) {
-            $this->moduleUpdater->setModuleOnDiskFromAddons($name);
+            if (!$this->moduleUpdater->setModuleOnDiskFromAddons($name)) {
+                throw new Exception($this->translator->trans('The module %name% has not been found on Addons.', ['%name%' => $name]));
```

Comment:
What domain is taken here if you do not specify 'Admin.Modules.Notification' ?

Reply:
I'm waiting Team Wording ;). I don't find any correct wording.